docs/en/stack/ml/df-analytics/ml-trained-models.asciidoc

```diff
@@ -13,16 +13,223 @@ information about this process, see <<ml-supervised-workflow>> and
+         "create_time":1604579862340,
+         "allow_lazy_start":false,
+         "description":"",
+         "analyzed_fields":{
+            "excludes":[
+            ],
+            "includes":[
+               "AvgTicketPrice",
+               "Carrier",
+               "Dest",
+               "DestCityName",
+               "DestCountry",
+               "DestWeather",
+               "DistanceKilometers",
+               "DistanceMiles",
+               "FlightDelay",
+               "FlightDelayMin",
+               "FlightTimeHour",
+               "FlightTimeMin",
+               "Origin",
+               "OriginCityName",
+               "OriginCountry",
+               "OriginWeather",
+               "dayOfWeek"
+            ]
+         },
+         "id":"reg-trained-model",
+         "source":{
+            "query":{
+               "match_all":{
+               }
+            },
+            "index":[
+               "kibana_sample_data_flights"
+            ]
+         },
+         "dest":{
+            "index":"reg-trained-model-ind",
+            "results_field":"ml"
+         },
+         "analysis":{
+            "regression":{
+               "randomize_seed":-5746203410061298773,
+               "dependent_variable":"FlightDelayMin",
+               "training_percent":10.0,
+               "loss_function":"mse",
+               "num_top_feature_importance_values":0,
+               "prediction_field_name":"FlightDelayMin_prediction"
+            }
+         },
+         "version":"7.9.0"
+      }
+   },
+   "input":{
+      "field_names":[
+        "AvgTicketPrice",
+        "Carrier",
+        "Dest",
+        "DestCityName",
+        "DestCountry",
+        "DestWeather",
+        "DistanceKilometers",
+        "DistanceMiles",
+        "FlightDelay",
+        "FlightDelayMin",
+        "FlightTimeHour",
+        "FlightTimeMin",
+        "Origin",
+        "OriginCityName",
+        "OriginCountry",
+        "OriginWeather",
+        "dayOfWeek"
+      ]
+   },
+   "inference_config":{
+      "regression":{
+         "results_field":"FlightDelayMin_prediction",
+         "num_top_feature_importance_values":0
+      }
+   }
+}
+--------------------------------------------------
+// TEST[skip:setup kibana sample data]
+====
+
+The API response contains the model information with metadata.
+--
+
+Your trained model is ready to be used as a <<ml-inference-processor,processor>> 
+in an ingest pipeline or as an <<ml-inference-aggregation,aggregation>>.
+
+[NOTE]
+--
+The trained model definition can be so large that it may take a long time for a 
+computer clipboard to copy and paste it. It is recommended to do it 
+programmatically, for example via a bash script or via Client code. You can find 
+examples below.
+--
+
+// Bash and Python examples
+
+[discrete]
+[[move-trained-model-to-es]]
+=== Moving a model to {es}
+
+It is possible to add a model to your {es} cluster even if the model is not 
+trained by {dfanalytics}.
```

Comment:
If I'm understanding correctly and this is about moving a model generated outside Elastic into Elastic, I think it might be clearer by adding this qualification:
```suggestion
trained by Elastic {dfanalytics}.
```